docs/source/configuration/index.rst

```diff
@@ -3,3 +3,94 @@
 =============
 Configuration
 =============
+
+You can customize the noise in the datsets the pseudopeople package generates.
+This allows you to explore different scenarios and see how sensitive entity resolution methods
+are to the types and levels of noise present in their input data.
+
+Overriding defaults
+-------------------
+
+Noise is configurable at a very fine-grained level.
+It can be customized separately for each **dataset** and **noise type**.
+:ref:`Column-based noise types <categories_of_noise>` can additionally have different settings for each **column**.
+
+Due to this fine-grained control, there are a very large number of settings.
+**It is not necessary to configure everything.**
+pseudopeople includes reasonable default noise settings
+and your configuration can override as few or as many of the default values as you like.
+To learn more about the default settings, see :ref:`Noise Type Details <noise_type_details>`.
+You can access the defaults from your Python code by calling the :func:`pseudopeople.get_config` function.
+
+Configuration structure
+-----------------------
+
+Configuration can be supplied as a nested Python dictionary, or as a YAML file.
+In either case, the structure is the same:
+
+* The top-level keys are the datasets.
+* Within each of these are keys for the :ref:`categories of noise <categories_of_noise>`: row-based and column-based.
+* **For column-based noise-only**, the next layer of keys is for the columns in the dataset.
+* Nested within these are keys for the individual :ref:`noise types <available_noise_types>`.
+* Finally, each noise type has parameters.
+
+As an example, say we wanted to change the cell probability parameter (which is the probability of a cell being wrong)
+of the :ref:`Choose the wrong option <choose_the_wrong_option>` noise type, for the sex column of the decennial Census dataset.
```

Comment:
Sorry if this is nit-picky, but I think if you don't capitalize 'decennial' you don't need to capitalize 'census' (for example you can see this grammar being used on the [Census Bureau's website](https://www.census.gov/programs-surveys/geography/about/glossary.html#:~:text=boundaries%20before%20each-,decennial%20census,-.%20Each%20ANRC%20is))

Reply:
Thanks for reviewing so closely! You are totally right!

Reply:
I think I'll capitalize both since it feels a little more "official" (i.e. it's specifically the US' census, not some other country's).